client/lib/domains/registration/availability-messages.js

```diff
@@ -146,10 +151,20 @@ function getAvailabilityNotice( domain, error, site ) {
 			break;
 		case domainAvailability.MAINTENANCE:
 			if ( tld ) {
+				let maintenanceEnd = translate( 'shortly', {
+					context: 'If a specific maintenance end time is unavailable, we will show this instead.',
```

Comment:
`comment` instead of `context` here too?

Reply:
Yep...somehow I always get those mixed up.